Av1an/VMAF/target_vmaf.py

```diff
@@ -210,8 +211,8 @@ def weighted_search(num1, vmaf1, num2, vmaf2, target):
     :return: Q for new probe
     """
 
-    dif1 = abs(target - vmaf2)
-    dif2 = abs(target - vmaf1)
+    dif1 = abs(-ln(1-target/100) - (-ln(1-vmaf2/100)))
+    dif2 = abs(-ln(1-target/100) - (-ln(1-vmaf1/100)))
```

Comment:
@krakow10 
`vmaf1` and `vmaf2` could be 100, in which case you can't take the log. Can you add something to protect against this? Something like this should work
```
# ensure vmaf != 100, to prevent ln(0)
vmaf1 = min(vmaf1, 99.99)
vmaf2 = min(vmaf2, 99.99)
```